lib/json_web_token/algorithm/hmac.ex

```diff
@@ -18,7 +18,7 @@ defmodule JsonWebToken.Algorithm.Hmac do
   """
   def sign(sha_bits, shared_key, signing_input) do
     validate_params(sha_bits, shared_key)
-    :crypto.hmac(sha_bits, shared_key, signing_input)
+    :crypto.mac(:hmac, sha_bits, shared_key, signing_input)
```

Comment:
Hi @lanodan 

what about to have the code like that:

```elixir
  if Code.ensure_loaded?(:crypto) and function_exported?(:crypto, :mac, 4) do
    defp hmac(type, key, data), do: :crypto.mac(:hmac, type, key, data)
  else
    defp hmac(type, key, data), do: :crypto.hmac(type, key, data)
  end
```

and then just call:

```elixir
hmac(sha_bits, shared_key, signing_input)
```


https://github.com/elixir-ecto/postgrex/blob/master/lib/postgrex/scram.ex#L69


and it's definitely would be great to merge it as soon as possible. but it looks like the library has no active support.